samples/csharp_webapi/13.core-bot/Controllers/BotController.cs

```diff
@@ -58,6 +58,8 @@ static BotController()
             _dialog = new MainDialog(bookingRecognizer, bookingDialog, _loggerFactory.CreateLogger<MainDialog>());
         }
 
+        [HttpPost]
+        [HttpGet]
         public async Task<HttpResponseMessage> Post()
```

Comment:
As long as we're touching this file, we should consider adding the `Async` suffix here.

Reply:
Sounds good, I'm betting it was left off before in order to take advantage of the auto-verb-matching, but now that we're explicitly adding verb tags it shouldn't be needed.